ivy_tests/test_ivy/test_frontends/test_numpy/test_matrix/test_methods.py

```diff
@@ -218,5 +264,6 @@ def test_numpy_matrix_any(
             "axis": axis,
         },
```

Comment:
an extension of the above comment, eg, `"dtype': dtype`

PS: If I'm not mistaken the return is already a single dtype with `full=False`, but just in case it is a list or tuple, just use `dtype[0]`

Reply:
both return lists